csrc/scheduler/reduction.cpp

```diff
@@ -762,6 +813,14 @@ std::shared_ptr<ReductionParams> outerReductionHeuristic(
         new_grdim * gidim % device_multiprocessor_count >
             grdim * gidim % device_multiprocessor_count) {
       grdim = new_grdim;
+      round_down = true;
+    }
+    // if can't round down, try fill empty SM slots of the last wave
+    if (!round_down) {
```

Comment:
isn't this just a `else`? We can try to put the `//try round waves down` and `// if can't round down` inside the block body for comment.

Reply:
you are right! changed.